PresentData/PresentMonTraceConsumer.cpp

```diff
@@ -216,10 +227,19 @@ void PMTraceConsumer::HandleDxgkBlt(EVENT_HEADER const& hdr, uint64_t hwnd, bool
     // yet, so PresentMode!=Unknown implies we looked up a 'stuck' present
     // whose tracking was lost for some reason.
     auto eventIter = FindOrCreatePresent(hdr);
+
+    if (eventIter == mPresentByThreadId.end()) {
+        return;
+    }
+
     if (eventIter->second->PresentMode != PresentMode::Unknown) {
         HandleStuckPresent(hdr, &eventIter);
     }
 
+    if (eventIter == mPresentByThreadId.end()) {

```

Comment:
Can you move this (and few other instances below) into the scope that calls HandleStuckPresent()?

Reply:
Good feedback - done!